src/Host/UserProfileInterface/Impl/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
```

Comment:
Should not have this. See other projects on a) AssemblyInfo and linked GlobalAssemblyInfo.cs - they allow build process to inject versions